### ISSUE TITLE
Master build does not include parquet

### ISSUE THREAD

**Issue by D0d0** [andypetrella/spark-notebook#750]
I used build to generate new notebook in [spark-notebook.io](spark-notebook.io) with this configuration
```
Build (retry #0)
•	notebook version : master
•	scala version : 2.11.8
•	spark version : 2.0.1
•	spark nightly : false
•	hadoop version : 2.7.2
•	with hive : true
•	with parquet : true
•	package type : zip
```
but downloaded folder does not include parquet. The folder name is ``spark-notebook-0.8.0-SNAPSHOT-scala-2.11.8-spark-2.0.1-hadoop-2.7.2-with-hive`` instead of ``spark-notebook-0.8.0-SNAPSHOT-scala-2.11.8-spark-2.0.1-hadoop-2.7.2-with-hive-with-parquet``. Also I checked application footer and there is no ``withParquet-true``.

Then I builded ``spark-notebook-0.6.3-scala-2.11.7-spark-1.6.2-hadoop-2.7.3-with-hive-with-parquet`` and checked application footer and there is ``withParquet-true``.

**Comment by andypetrella** [comment]
would you have an URL to check what's in?

The one provided in the mail would be okay

**Comment by D0d0** [comment]
https://s3.eu-central-1.amazonaws.com/spark-notebook/zip/spark-notebook-master-scala-2.11.8-spark-2.0.1-hadoop-2.7.2-with-hive.zip?max-keys=100000

**Comment by andypetrella** [comment]
Mmmmh sure? In the `lib` folder I see

```
org.apache.parquet.parquet-column-1.7.0.jar
org.apache.parquet.parquet-common-1.7.0.jar
org.apache.parquet.parquet-encoding-1.7.0.jar
org.apache.parquet.parquet-format-2.3.0-incubating.jar
org.apache.parquet.parquet-generator-1.7.0.jar
org.apache.parquet.parquet-hadoop-1.7.0.jar
org.apache.parquet.parquet-jackson-1.7.0.jar
```

**Comment by D0d0** [comment]
Yes, you are right, I tried `import org.apache.parquet` and it is impored (my bad that i did not try it before :/).

But `withParquet-true` is missing in application footer and also missing in folder which can confuse somebody as me.

**Comment by vidma** [comment]
I think we removed this option , because `parquet` is included by default since spark `2.0.x`
